### PR TITLE
Add info about new contraction data type and permutation feature to README

### PR DESCRIPTION
**Pull request by CongMa13**
Add info about new contraction data type and permutation feature to README

### CODE REVIEW

File: README.md
Comment (LisaDelaney) [inline README.md]: This should be sentence case:

```suggestion
## Running permutation tests
```

File: README.md
Comment (LisaDelaney) [inline README.md]: I don't think we need this heading.

File: README.md
Comment (LisaDelaney) [inline README.md]:
```suggestion
Test API implementation of the permutation algorithm with validation.
```

File: README.md
Comment (LisaDelaney) [inline README.md]:
```suggestion
### F32 bilinear contraction
```

File: README.md
Comment (LisaDelaney) [inline README.md]:
```suggestion
### F32 scale contraction
```

File: README.md
Comment (LisaDelaney) [inline README.md]: How about: "The client application links to hipTensor library; therefore, you must install the hipTensor library before building client applications."

File: README.md
Comment (LisaDelaney) [inline README.md]:
```suggestion
## Running contraction tests
```

File: README.md
Comment (LisaDelaney) [inline README.md]: Can we make this heading (and Scale contraction tests) into bullet points in order to avoid stacked headings?

**Comment by CongMa13** [comment]
Hi @LisaDelaney , I have made changes you suggested. Could you review it again?

File: README.md
Comment (LisaDelaney) [inline README.md:41]: Does "Contractor Tensor" need to be capitalized (i.e. is it a formal name)? All titles should be in sentence case.

Comment (LisaDelaney) [inline README.md]:
```suggestion
### Operation: Contraction tensor
```

File: README.md
Comment (LisaDelaney) [inline README.md]:
```suggestion
### Operation: Permutation tensor
```

File: README.md
Comment (LisaDelaney) [inline README.md]:
```suggestion
After configuration, build with `cmake --build <build_dir> -- -j<nproc>`.
```

File: README.md
Comment (LisaDelaney) [inline README.md]:
```suggestion
The client application links to the hipTensor library; therefore, you must install the
```

File: README.md
Comment (LisaDelaney) [inline README.md]:
```suggestion
### Build samples as external client
```

**Comment by CongMa13** [comment]
Hi @LisaDelaney , the changes you suggested have been merged. Could you review it again?

**Comment by CongMa13** [comment]
Hi @LisaDelaney , @saadrahim ,  could you please approve this PR if it looks good to you?  I can merge it after your approval. Thanks.

File: README.md
Comment (LisaDelaney) [inline README.md]: Can we make sure all titles have sentence case?
```suggestion
## Minimum software requirements
```

File: README.md
Comment (LisaDelaney) [inline README.md:48]: Should the letters in this table not be in caps? (e.g., BF16)

File: README.md
Comment (LisaDelaney) [inline README.md]: I would suggest changing "is supported only on" to "is only supported on".

File: README.md
Comment (CongMa13) [reply README.md:48]: I think we can keep the bf16 to be consistent with the RocWMMA's format. 

RocWMMA's readme https://github.com/ROCm/rocWMMA

image

File: README.md
Comment (CongMa13) [reply README.md]: changed to "is only supported on".

**Comment by CongMa13** [comment]
Hi @LisaDelaney ,  the README.md has been updated as you suggested. Could you approve this PR if it's ready to go?

**Review by LisaDelaney — DISMISSED** [review]

**Comment by CongMa13** [comment]
Hi @LisaDelaney , could you approve this PR again. 

Sorry to ask you to do this again. 

Deleted this line from README.md to make it to pass all the checks. 

```
hiptensor supports a tensor contraction of the form $D = \alpha AB + \beta C$
```

Remove it since readthedocs does not accept the Latex expression in this line.

**Comment by LisaDelaney** [comment]
> Hi @LisaDelaney , could you approve this PR again.
> 
> Sorry to ask you to do this again.
> 
> Deleted this line from README.md to make it to pass all the checks.
> 
> ```
> hiptensor supports a tensor contraction of the form $D = \alpha AB + \beta C$
> ```
> 
> Remove it since readthedocs does not accept the Latex expression in this line.

If you want to leave it in, I can find math syntax that works...just let me know.

**Review by LisaDelaney — APPROVED** [review]

**Review by mkarunan — APPROVED** [review]
lgtm

**Pull request merged**
